### PR TITLE
nodebrew make nodejs symlink

### PR DESCRIPTION
**Pull request by satmandu**
- nodejs needs to be a symlink.
- rubify

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] armv7
- [x] i686

### CODE REVIEW

File: packages/nodebrew.rb [hunk 1]
```diff
@@ -3,45 +3,62 @@
...
+  binary_sha256({
+    aarch64: 'e3c4bfac2033982be9859766dc9b7c00a018cbdc0c233f54b0725295baabd770',
+     armv7l: 'e3c4bfac2033982be9859766dc9b7c00a018cbdc0c233f54b0725295baabd770',
+       i686: 'ca1437cc074946cd723a691a89ca127d3c1f95beea6015fa74cf66fe074a6c22',
+     x86_64: 'bcf965ab509fca115094ba9ed146c0b9fe43a37055b2b233b32cf77bde659f08'
+  })
+
   def self.install
-    system "mkdir -p #{CREW_DEST_PREFIX}/share/nodebrew"
+    @home = ENV['HOME']
```
Comment (uberhacker) [inline packages/nodebrew.rb:26]: We already have the crew constant `HOME`.  You can use `#{HOME}` where needed.

File: packages/nodebrew.rb [hunk 2]
```diff
@@ -3,45 +3,62 @@
...
     system "NODEBREW_ROOT=#{CREW_DEST_PREFIX}/share/nodebrew perl nodebrew setup > /dev/null"
-    system "mkdir -p #{CREW_DEST_DIR}/$HOME"
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/nodebrew #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/node #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npm #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npx #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew #{CREW_DEST_DIR}/$HOME/.nodebrew"
-    system "ln -sf #{CREW_PREFIX}/share/nodebrew $HOME/.nodebrew"
+    FileUtils.mkdir_p "#{CREW_DEST_DIR}#{@home}"
```
Comment (uberhacker) [inline packages/nodebrew.rb:29]: `FileUtils.mkdir_p CREW_DEST_HOME`?

File: packages/nodebrew.rb [hunk 3]
```diff
@@ -3,45 +3,62 @@
...
-    puts "nodebrew use latest".lightblue
+    puts 'To install the latest node, execute:'.lightblue
+    puts 'nodebrew install-binary latest'.lightblue
+    puts 'nodebrew use latest'.lightblue
     puts
-    puts "To uninstall, execute the following:".lightblue
-    puts "crew remove nodebrew".lightblue
+    puts 'To uninstall, execute the following:'.lightblue
+    puts 'crew remove nodebrew'.lightblue
     puts "rm -rf #{CREW_PREFIX}/share/nodebrew".lightblue
```
Comment (uberhacker) [inline packages/nodebrew.rb:62]: Add in `def self.remove` section instead?

File: packages/nodebrew.rb [hunk 4]
```diff
@@ -3,45 +3,61 @@
...
     system "NODEBREW_ROOT=#{CREW_DEST_PREFIX}/share/nodebrew perl nodebrew setup > /dev/null"
-    system "mkdir -p #{CREW_DEST_DIR}/$HOME"
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/nodebrew #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/node #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npm #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew/current/bin/npx #{CREW_DEST_PREFIX}/bin/"
-    system "ln -s #{CREW_PREFIX}/share/nodebrew #{CREW_DEST_DIR}/$HOME/.nodebrew"
-    system "ln -sf #{CREW_PREFIX}/share/nodebrew $HOME/.nodebrew"
+    FileUtils.mkdir_p "#{CREW_DEST_DIR}#{HOME}"
```
Comment (uberhacker) [inline packages/nodebrew.rb:28]: `FileUtils.mkdir_p CREW_DEST_HOME`?

File: packages/nodebrew.rb (hunk 3 above)
Comment (satmandu) [reply packages/nodebrew.rb:61]: Is `def self.remove` automatically run? Because this should be optional, no?

File: packages/nodebrew.rb [hunk 5]
```diff
@@ -3,45 +3,61 @@
...
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/nodebrew", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/node", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/npm", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/npx", "#{CREW_DEST_PREFIX}/bin/"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew/current/bin/node", "#{CREW_DEST_PREFIX}/bin/nodejs"
+    FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew", "#{CREW_DEST_DIR}#{HOME}/.nodebrew"
```
Comment (uberhacker) [inline packages/nodebrew.rb:35]: `FileUtils.ln_s "#{CREW_PREFIX}/share/nodebrew", "#{CREW_DEST_HOME}/.nodebrew"`?

File: packages/nodebrew.rb (hunk 3 above)
Comment (satmandu) [reply packages/nodebrew.rb]: Does this look right?

**Pull request merged**